Packs/Automox/Integrations/Automox/Automox.yml

```diff
@@ -0,0 +1,1078 @@
+      description: 'Frequency of device refreshes in minutes'
+      type: Number
+    - contextPath: Automox.CreatedGroups.parent_server_group_id
+      description: 'Identifier of parent group'
+      type: Number
+    - contextPath: Automox.CreatedGroups.ui_color
+      description: 'Automox console highlight color for the group'
+      type: String
+    - contextPath: Automox.CreatedGroups.notes
+      description: 'Notes defined for the group'
+      type: String
+    - contextPath: Automox.CreatedGroups.enable_os_auto_update
+      description: 'Enable operating system auto updates'
+      type: Boolean
+    - contextPath: Automox.CreatedGroups.server_count
+      description: 'Number of devices assigned to group'
+      type: Number
+    - contextPath: Automox.CreatedGroups.policies
+      description: 'List of policies assigned to group'
+      type: Number
+  - name: automox-group-update
+    description: "Update a group's information in Automox"
+    arguments:
+    - name: org_id
+      description: 'Organization identifier'
+      type: 'number'
+      isArray: false
+      required: false
+      default: false
+      secret: false
+    - name: group_id
+      description: 'Group identifier'
+      type: 'number'
+      isArray: false
+      required: true
+      default: false
+      secret: false
+    - name: name
+      description: 'Name of the group'
+      type: String
+      isArray: false
+      required: false
+      default: false
+      secret: false
+    - name: notes
+      description: 'Define notes for the group'
+      type: String
+      isArray: false
+      required: false
+      default: false
+      secret: false
+    - name: parent_server_group_id
+      description: 'Identifier of the parent group. Defaults to default group id if omitted.'
+      type: Number
+      isArray: false
+      required: false
+      default: false
+      secret: false
+    - name: policies
+      description: 'List of policy identifiers to assign to group'
+      type: Number
+      isArray: true
+      required: false
+      default: false
+      secret: false
+    - name: refresh_interval
+      description: 'Frequency of device refreshes in minutes'
+      type: Number
+      isArray: false
+      required: false
+      default: false
+      secret: false
+    deprecated: false
+    execution: false
+  - name: automox-group-delete
+    description: "Delete a group from Automox"
+    arguments:
+    - name: org_id
+      description: 'Organization identifier'
+      type: Number
+      isArray: false
+      required: false
+      default: false
+      secret: false
+    - name: group_id
+      description: 'Group identifier'
+      type: Number
+      isArray: false
+      required: true
+      default: false
+      secret: false
+    deprecated: false
+    execution: false
+  isfetch: false
+  runonce: false
+  script: '-'
+  type: python
+  subtype: python3
+  dockerimage: demisto/python3:3.10.4.27798
+fromversion: 5.5.0
```

Comment:
Is there a reason for fromversion 5.5.0? as new contributions should be supported from 6.0.0 version

Reply:
There is no reason, I'm not sure why it was 5.5.0 instead of 6.0.0, it is updated now though. Done.